python/ray/serve/config.py

```diff
@@ -84,6 +91,33 @@ def replicas_settings_valid(cls, max_replicas, values):
 
         return max_replicas
 
+    @validator("policy", always=True)
+    def serialize_policy(cls, policy, values):
+        """Serialize policy to be cloudpickled and base64 encoded utf8 string.
+
+        If policy is a string, it is possible to be already serialized when constructed
+        from protobuf. In this case, we just return the already serialized string. Else
+        we import the policy and serialize it before return.
+        """
+        if isinstance(policy, Callable):
+            imported_policy = policy
+        elif isinstance(policy, str):
+            try:
+                assert b64encode(b64decode(policy)).decode("utf-8") == policy
+                return policy
+            except binascii.Error:
+                imported_policy = import_attr(policy)
+            except AssertionError:
+                imported_policy = import_attr(policy)
+        else:
+            return policy
+
+        return b64encode(cloudpickle.dumps(imported_policy)).decode("utf-8")
+
+    def get_policy(self) -> Callable:
+        """Deserialize policy from cloudpicked base64 encoded utf8 string."""
```

Comment:
```suggestion
        """Deserialize policy from cloudpickled, base64-encoded, utf8 string."""
```

Reply:
why is this base64 encoded as a string?

Reply:
Because base64 encode will return a byte object and I don't want to have handle 3 data types as the input for policy 🙏

Reply:
why not just use `bytes` directly?

Reply:
Because this config came from the users. The support for string and callable is required. Bytes are the optional one.

Reply:
My understanding is that users can pass a callable or an import path. In both of these cases, we will resolve it to cloudpickled bytes of the actual class definition.

In that case, the type of this field should be `bytes` and the layer above can resolve from `Union[str, Callable]` into bytes when the config object is constructed.

Am I missing something?

Reply:
Yea, good idea! Can make a public `policy` for users to pass in the scaling function, and have a private `policy_def` or something that's in bytes.